cli/fathom_web/commands/train.py

```diff
@@ -8,7 +8,7 @@
 from torch.optim import Adam
 
 from ..accuracy import accuracy_per_tag, accuracy_per_page, pretty_accuracy
-from ..utils import classifier, mini_histogram, tensors_from
+from ..utils import classifier, mini_histogram, speed_readout, tensors_from
```

Comment:
`mini_histogram` is no longer used here.

Reply:
Thanks! In test.py, too, it turns out.